rollup/internal/controller/watcher/chunk_proposer_test.go

```diff
@@ -568,51 +568,53 @@ func testChunkProposerCodecv2Limits(t *testing.T) {
 }
 
 func testChunkProposerBlobSizeLimit(t *testing.T) {
-	compressionTests := []bool{false, true} // false for uncompressed, true for compressed
-	for _, compressed := range compressionTests {
+	codecVersions := []encoding.CodecVersion{encoding.CodecV0, encoding.CodecV1, encoding.CodecV2}
+	for _, codecVersion := range codecVersions {
 		db := setupDB(t)
-		block := readBlockFromJSON(t, "../../../testdata/blockTrace_02.json")
-		for i := int64(0); i < 2000; i++ {
+		block := readBlockFromJSON(t, "../../../testdata/blockTrace_03.json")
+		for i := int64(0); i < 510; i++ {
 			l2BlockOrm := orm.NewL2Block(db)
 			block.Header.Number = big.NewInt(i + 1)
 			err := l2BlockOrm.InsertL2Blocks(context.Background(), []*encoding.Block{block})
 			assert.NoError(t, err)
 		}
 
 		var chainConfig *params.ChainConfig
-		if compressed {
-			chainConfig = &params.ChainConfig{BernoulliBlock: big.NewInt(0), CurieBlock: big.NewInt(0)}
-		} else {
+		if codecVersion == encoding.CodecV0 { // will never hit blob size limit
+			chainConfig = &params.ChainConfig{}
+		} else if codecVersion == encoding.CodecV1 {
 			chainConfig = &params.ChainConfig{BernoulliBlock: big.NewInt(0)}
+		} else {
+			chainConfig = &params.ChainConfig{BernoulliBlock: big.NewInt(0), CurieBlock: big.NewInt(0)}
 		}
 
 		cp := NewChunkProposer(context.Background(), &config.ChunkProposerConfig{
-			MaxBlockNumPerChunk:             math.MaxUint64,
+			MaxBlockNumPerChunk:             255,
```

Comment:
da-codec has added a max 255 blocks per chunk check, thus fixing this unit test.